MapleServer2/PacketHandlers/Game/VibrateHandler.cs

```diff
@@ -26,7 +26,7 @@ public override void Handle(GameSession session, PacketReader packet)
             int unk2 = packet.ReadInt();
             int unk3 = packet.ReadInt();
 
-            session.FieldManager.BroadcastPacket(VibratePacket.Vibrate(entityId, someId, objectId, flag, session.Player, session.ClientTick));
+            session.FieldManager.BroadcastPacket(VibratePacket.Vibrate(entityId, someId, objectId, flag, session.FieldPlayer, session.ServerTick));
```

Comment:
Why did the player need to become FieldPlayer?

Reply:
session.Player's coordinates only get updated on map enter/exit so using the field player coordinates is more accurate.